phpstan-config.neon

```diff
@@ -1,6 +1,11 @@
 parameters:
-    level: 1
+    level: 5
```

Comment:
is there anything blocking from using higher levels ?

Reply:
Nope, I have choosen this more or less random and started fixing.

Increasing it further could be subject for a new PR

Reply:
well, I would suggest to try running it at max level (`8`) and see whether it finds anything else. If it is already green at max level with the changes done here, there is no reason to wait for another PR.

Reply:
justed tested level max and level 6.

as of level 6 phpstan requires all properties to be typed.

its not so much work but I dont want to invest it now.